Automattic/jetpack
# Boost: Update Purchase Successful copy

Fixes https://github.com/Automattic/jetpack/issues/33983

## Proposed changes:

* Increase section width and spacings to better match designs;
* Update Purchase Successful copy (heading text, listing text, image, added link);
* Update Upgrade page spacings (to match Purchase Successful).

<details><summary>Purchase Successful before:</summary>

![CleanShot 2024-01-22 at 15 47 15](https://github.com/Automattic/jetpack/assets/11799079/b6ffae9e-7941-446f-bb9f-5c4e1575f49c)


</details>

<details><summary>Purchase Successful now:</summary>

![image](https://github.com/Automattic/jetpack/assets/11799079/563e66e3-a91a-4e26-93ab-85e00c91c4d8)

</details>

### Other information:


## Jetpack product discussion



pc9hqz-2g5-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Check `/wp-admin/admin.php?page=jetpack-boost#purchase-successful` to make things match the designs;
* Also make sure that nothing's broken 😅
